From 7dee972d3674c69c17d960fa64265db4430475fb Mon Sep 17 00:00:00 2001
From: Steven Noonan <steven@uplinklabs.net>
Date: Fri, 16 Sep 2016 10:15:49 -0700
Subject: [PATCH] xhci: demote annoying warning

Smart cards are always spewing this warning and it's getting on my
nerves.

Signed-off-by: Steven Noonan <steven@uplinklabs.net>
---
 drivers/usb/host/xhci-ring.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/host/xhci-ring.c b/drivers/usb/host/xhci-ring.c
index fd9fd12..f51cead 100644
--- a/drivers/usb/host/xhci-ring.c
+++ b/drivers/usb/host/xhci-ring.c
@@ -2463,7 +2463,7 @@ static int handle_tx_event(struct xhci_hcd *xhci,
 			 */
 			if (!(trb_comp_code == COMP_STOP ||
 						trb_comp_code == COMP_STOP_INVAL)) {
-				xhci_warn(xhci, "WARN Event TRB for slot %d ep %d with no TDs queued?\n",
+				xhci_dbg(xhci, "WARN Event TRB for slot %d ep %d with no TDs queued?\n",
 						TRB_TO_SLOT_ID(le32_to_cpu(event->flags)),
 						ep_index);
 				xhci_dbg(xhci, "Event TRB with TRB type ID %u\n",
-- 
2.9.3

